DOCS-1089: Update Modular resource examples to use new UI (#1806)
viamrobotics/docs: docs/extend/modular-resources/examples/csi.md

## Configuration

{{< tabs name="Connect your CSI Module and Modular Resource">}}
{{% tab name="Config Builder" %}}

Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).

Click on the **Modules** subtab and navigate to the **Local** section.

> Enter a name, for example `my_csi_cam_module_name`, and the executable path, for example `/usr/bin/csi-mr`.

Reviewer: ```suggestion
Enter a name, for example `my_csi_cam_module_name`, then enter the executable path.
If you used the above install command, the executable path should be: `/usr/local/bin/viam-csi`.
```

Then click **Add module**.

Click on the **Components** subtab and click **Create component**.

> Select the `local modular resources` type.

Reviewer: ```suggestion
Select the `local modular resource` type.
```
To match UI

> Then select the `camera` as the type, enter the triplet `viam:camera:csi` and give your resource a name, for example `my_test_csi_cam`.

Reviewer: ```suggestion
Then select `camera` as the type, enter the triplet `viam:camera:csi`, and give your resource a name, for example `my_test_csi_cam`.
```

Click **Create**.

On the new component panel, copy and paste the following JSON object into the attributes field:

```json
{
    "width_px": <int>,
    "height_px": <int>,
    "frame_rate": <int>,
    "debug": "<boolean>"
}
```

{{% /tab %}}
{{% tab name="JSON Template" %}}

Go to your robot's page on the [Viam app](https://app.viam.com/).
Navigate to the **Config** tab on your robot's page and select **Raw JSON** mode.

Copy and paste the JSON object for the module into the modules array to add Viam's `csi-mr` module:

```json
{

>     "executable_path": "</usr/bin/csi-mr>",

Reviewer: ```suggestion
    "executable_path": "</usr/local/bin/viam-csi>",
```
